### ISSUE TITLE
[VT-1] clear "Project:" after escape

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#23]
_Scott Kostyshak on 2013-04-06T21:38:24Z says:_ 

 If I press 'p', "project:" appears at the bottom. If I press "escape", Vit correctly forgets about project and is ready for a new command. However, "project:" is still left at the bottom.

This is not the case for 'h' (set priority to high), 'a' (add task), or 'd' (mark task as done).

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-04-06T21:38:24Z
Modified: 2014-02-09T01:14:57Z
```

**Comment by taskwarrior** [comment]
_steve rader on 2013-04-06T21:57:43Z says:_ 

 well, yeah, that's not right, and should be fixed.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-07-07T00:52:46Z says:_ 

 I think the problem is that getch_loop is not being called in this situation.

See the part 

      if ( $ch eq "\e" ) {
        $error_msg = '';
        $feedback_msg = '';
        $refresh_needed = 1;
        $input_mode = 'cmd';
        last CASE;
      }

which appears to do the right thing when given the chance.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-10-04T21:51:48Z says:_ 

 I will look at this for 1.2 since I'm looking at key codes. Should be a simple fix.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-10-07T03:12:30Z says:_ 

 Fixed at a984e607. It takes a while for the escape to clear. This is probably because getch is making sure that the key sequence is terminated.
